monitoring/oplog-populator/alerts.yaml

```
@@ -51,3 +51,14 @@ groups:
       description: "Oplog populator metastore change stream lag is too big"
       summary: "Oplog populator configuration lag is above threshold"
 
+  - alert: KafkaConnectFailedConnectors
+    Expr: |
+        sum by(connector) (increase(s3_oplog_populator_connector_restarts{job="${oplog_populator_job}",namespace="${namespace}"}[5m]))
+        > 0
+    For: "1m"
```

timing should probably be the other way:

```suggestion
    Expr: |
        sum by(connector) (increase(s3_oplog_populator_connector_restarts{job="${oplog_populator_job}",namespace="${namespace}"}[1m]))
        > 0
    For: "10m"
```

→ check that we restart every minute in the last 10 minutes (instead of checking for 1min that we restarted in the last 5 min)